### PR TITLE
allow public import for bookmarks (both from browser and from file)

### PR DESCRIPTION
**Pull request by martinraison**
@stkem @2is10 ( @andrewconner )

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html [hunk 1]
```diff
@@ -29,27 +29,27 @@ <h2 class="kf-footer-header">Friends on Kifi</h2>
...
+          <span class="sprite sprite-import-bookmarks-icon"></span><a href="javascript:">Import your bookmarks</a>
+        </div>
       </li>
       <li class="import-bookmark-file">
-        <a class="kf-footer-item-link" href="javascript:" ng-click="importBookmarkFile()">
-          <span class="sprite sprite-import-from-other-services-icon"></span>Import Delicious/Kippt/Pocket/...
-        </a>
+        <div class="kf-footer-item-link" ng-click="importBookmarkFile()">
+          <span class="sprite sprite-import-from-other-services-icon"></span><a href="javascript:">Import Delicious/Kippt/Pocket/...</a>
+        </div>
```
Comment (stkem) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html:52]: Why this change? The icon will be no longer clickable.

**Comment by stkem** [comment]
made one small comment. lgtm otherwise

File: kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html (hunk 1 above)
Comment (martinraison) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html:52]: it will be, since the whole div is clickable (and the cursor style is set to "pointer"). I just wanted to fill the "hole" between the icon and the link that was not clickable, as per Jared's advice.

Comment (stkem) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html:52]: ahh, good point!

Comment (2is10) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html:52]: If I remember correctly, all that was needed was to add `display: inline-block` on the `a` elements. (Remember, we tried it at your desk?)

We should generally try to avoid adding more elements. Also, making click targets out of divs is generally frowned upon because they’re less accessible (generally can’t be tabbed to and screen readers won’t recognize them as click targets).

Comment (martinraison) [inline kifi-backend/modules/shoebox/angular-black/src/layout/rightCol/rightCol.tpl.html:52]: fixed in #9198 - thanks for the comment!
